lib/features/mailbox/presentation/mailbox_controller.dart

```diff
@@ -152,7 +153,8 @@ class MailboxController extends BaseMailboxController {
     newState.map((success) async {
       if (success is GetAllMailboxSuccess) {
         _currentMailboxState = success.currentMailboxState;
-        await buildTree(success.mailboxList);
+        final _mailboxList = success.mailboxList.where((mailbox) => mailbox.isSubscribed?.value == true).toList();
```

Comment:
should separate it in method for more understand, no need comment but use method name for that :)